deployments/rialto/docker-compose.git.yml

```diff
@@ -46,22 +46,22 @@ services:
         BRIDGE_HASH: ${RELAY_BRIDGE_HASH:-master}
         PROJECT: ethereum-poa-relay
 
-  bridge-node-alice: &bridge-node
+  rialto-bridge-node-alice: &rialto-bridge-node
     build:
       context: .
       dockerfile: ./Bridge.Dockerfile
       args:
         BRIDGE_HASH: ${NODE_BRIDGE_HASH:-master}
-        PROJECT: bridge-node
+        PROJECT: rialto-bridge-node
 
-  bridge-node-bob:
-    <<: *bridge-node
+  rialto-bridge-node-bob:
```

Comment:
These names need the `rialto` prefix removed. Otherwise when we merge the two files (if we want to use the Git one for instance) Docker doesn't realize that we want to override nodes and we end up with a ton of duplicated nodes